src/main/java/com/chat/yourway/service/EmailSenderService.java

```diff
@@ -17,26 +20,28 @@
 @EnableAsync
 @RequiredArgsConstructor
 public class EmailSenderService {
-    @Value("${spring.mail.username}")
-    private String emailAddressFrom;
-    private final JavaMailSender javaMailSender;
-
-    @Async
-    public void sendEmail(EmailSend request) {
-        if (Objects.isNull(request))
-            return;
-
-        MimeMessage mail = javaMailSender.createMimeMessage();
-        MimeMessageHelper helper = new MimeMessageHelper(mail);
-
-        try {
-            helper.setFrom(emailAddressFrom);
-            helper.setTo(request.to());
-            helper.setSubject(request.subject());
-            helper.setText(request.text());
-            javaMailSender.send(mail);
-        } catch (MessagingException e) {
-            throw new RuntimeException(e);
-        }
+
+  @Value("${spring.mail.username}")
+  private String emailAddressFrom;
+  private final JavaMailSender javaMailSender;
+
+  @Async
+  public void sendEmail(EmailSend request) {
+      if (Objects.isNull(request)) {
```

Comment:
Can it be null? Also read javadoc of Objects - it shouldn`t be used in this way as you used here